src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -152,6 +152,12 @@ public double getAngle() {
 
 	public void setAngle(double angle) {
 		launcherAnglePIDController.setReference(Units.degreesToRotations(angle), ControlType.kPosition);
+		System.out.println("Setting Angle!");
+	}
+
+	public boolean isAtTargetSpeed() {
+		return (setLauncherSpeedEntry.getDouble(0)
+				== launcherSpeedEntry.getDouble(SPEAKER_SHOOT_SPEED_RPM));
```

Comment:
Would prefer to get the velocity directly from the encoder, instead of relying on `launcherSpeedEntry` to always be updated.

Also, if we are using `setLauncherSpeedEntry` to store the target speed, we should update it in `launch(speed)`, otherwise this will return false when we use that method.

Reply:
1. bet


2.launch(speed) is mainly for presets, if they set a speed before a match, then it'll get messed up during auto.